snapcraft.yaml

```diff
@@ -0,0 +1,19 @@
+name: tqdm
+version: '1.0'
+summary: A fast, extensible progress bar for Python and CLI
+description: |
+  A fast, extensible progress bar for Python and CLI
```

Comment:
any way to inject a file (README.rst) into this field?

Reply:
You could perhaps have a script that generates a snapcraft.yaml stub with description already pre-populated, and then add the remaining stanzas. Or maybe pipe readme.rst into sed and then replace a default description with yours.